### PR TITLE
67 api reference for one shot device

### PR DESCRIPTION
**Pull request by jacekroszkowiakdev**
Please check the docs and diagram. I have also made a few changes in the slu-scripts readme.

### CODE REVIEW

File: docker-compose.yml.BACKUP [hunk 1]
```diff
@@ -0,0 +1,122 @@
+version: "3.7"
```
Comment (dominic22) [inline docker-compose.yml.BACKUP:1]: Can this file be removed?

Comment (jacekroszkowiakdev) [reply docker-compose.yml.BACKUP]: Thanks - I have missed that one. File removed from git cache.

**Comment by dominic22** [comment]
Please add an assets folder and move the `one-shot-microservice-diagram.png` into it

**Comment by jacekroszkowiakdev** [comment]
assets directory created and one-shot-microservice-diagram.png into it

**Review by dominic22 — APPROVED** [review]

**Pull request merged**